doc/classes/KinematicBody2D.xml

```diff
@@ -98,14 +110,15 @@
 			<argument index="1" name="rel_vec" type="Vector2">
 			</argument>
 			<description>
-				Returns true if there would be a collision if the body moved from the given point in the given direction.
+				Checks for collisions without moving the body. Virtually sets the node's position, scale and rotation to that of the given [Transform2D]. Then tries to move the body along the vector [code]rel_vec[/code]. Returns true if it encounters any obstacle.
```

Comment:
"Then tries..." is not a complete sentence. This can be a comma:  "given [Transform2D], then tries"

Last sentence: "Returns [code]true[/code] if a collision would occur."